pkg/engine/jsonContext.go

```diff
@@ -350,16 +351,14 @@ func fetchConfigMap(logger logr.Logger, entry kyvernov1.ContextEntry, ctx *Polic
 		namespace = "default"
 	}
 
-	obj, err := ctx.Client.GetResource("v1", "ConfigMap", namespace.(string), name.(string))
+	obj, err := ctx.InformerCacheResolvers.Get(context.Background(), namespace.(string), name.(string))
```

Comment:
`context.TODO()` we will need to propagate the context in the near future.

Reply:
fixed